gencert.sh

```diff
@@ -11,3 +20,19 @@ docker run --rm --volume $PWD/certs:/tmp --user $(id -u):$(id -g) eventstore/es-
 docker run --rm --volume $PWD/certs:/tmp --user $(id -u):$(id -g) eventstore/es-gencert-cli:1.0.2 create-node -ca-certificate /tmp/ca/ca.crt -ca-key /tmp/ca/ca.key -out /tmp/node -ip-addresses 127.0.0.1 -dns-names localhost
 
 chmod -R 0755 ./certs
+
+echo ">> Copying certificate..."
+cp certs/ca/ca.crt /usr/local/share/ca-certificates/eventstore_ca.crt
+
+if [ "${machine}" == "MacOS" ]; then
+  echo ">> Installing certificate on ${machine}..."
+  sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain /usr/local/share/ca-certificates/eventstore_ca.crt   
+elif [ "$machine" == "Linux" ]; then
+  echo ">> Installing certificate on ${machine}..."
+  sudo update-ca-certificates    
+elif [ "$machine" == "WSL" ]; then
```

Comment:
Since they both run the same command, can we merge the Linux and WSL blocks?

Reply:
👍